cinema/serializers.py

```diff
@@ -79,15 +116,18 @@ class Meta:
             "cinema_hall_name",
             "cinema_hall_capacity",
             "tickets_available",
+            "movie_image",
         )
 
 
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], attrs["seat"], attrs["movie_session"]
-        )
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"])
```

Comment:
In the `validate` method of `TicketSerializer`, ensure that `attrs` contains the necessary keys (`row`, `seat`, `movie_session`) before accessing them to prevent potential `KeyError`. Consider using `attrs.get('key')` with default values or handling the exception.